tests/functional/fixtures/model_download_fixtures.py

```diff
@@ -43,6 +45,24 @@ def download_model(model_url_base, model_name, model_version, dir):
     return local_bin_path, local_xml_path
 
 
+def download_data(data_url_base, model_name, model_version, dir):
+    local_data_path = os.path.join(dir, model_name, model_version, "data")
+    
+    local_tar_path = os.path.join(local_data_path,
+                                  "{}.{}".format(model_name, "tar.gz"))
+    if not os.path.exists(local_data_path):
+        print("Downloading " + model_name + " data...")
+        print(dir)
+        os.makedirs(local_data_path)
+        response = requests.get(data_url_base, stream=True)
+        with open(local_tar_path, 'wb') as output:
+            output.write(response.content)
+    
+    tf = tarfile.open(local_tar_path)
```

Comment:
it's just style comment, but I think `tf` shorthand could be confusing. As far as I know `tf` is very popular with TensorFlow.